app/views/api/guidance/release_notes.yml

```diff
@@ -0,0 +1,47 @@
+# - title: This is an example note
```

Comment:
So I guess you went with yaml for the structure, but I wonder if we end up tying our hands if we need more custom stylings, maybe its ok for now

Reply:
Yeah, markdown is supported here though - not sure what extra styling we might need in a release note.